include/openspace/rendering/framebufferrenderer.h

```diff
@@ -59,6 +59,89 @@ class FramebufferRenderer final : public RaycasterListener, public Deferredcaste
 public:
     virtual ~FramebufferRenderer() override = default;
 
+    //============================//
+    //=====  Reuse textures  =====//
+    //============================//
+    /**
+     * Gives access to the currently NOT used pingPongTexture. This texture is available
+     * for all RenderBins. However, it cannot be used at the same time as the Deferred
+     * Caster Tasks. The size of the texture is the resolution of the viewport.
+     *
+     * NOTE (malej 2023-FEB-21): The currently NOT used pingPongTexture might change
+     * depending on where in the render cycle you are. Especially after the Deferred
+     * Caster Tasks.
+     *
+     * \return GLuint identifier of the currently NOT used pingPongTexture
```

Comment:
No need for the `GLuint` here;  the `\return` call only takes text as an argument